src/cavendish_particle_tracks/_magnification_dialog.py

```diff
@@ -30,77 +38,99 @@ def __init__(self, parent=None):
         self.b1 = Fiducial()
         self.b2 = Fiducial()
 
-        # drop-down lists of fiducials
-        self.cbf1 = self._setup_dropdown_fiducials_combobox()
-        self.cbf2 = self._setup_dropdown_fiducials_combobox()
-        self.cbb1 = self._setup_dropdown_fiducials_combobox(back=True)
-        self.cbb2 = self._setup_dropdown_fiducials_combobox(back=True)
+        # region UI Setup
+        self.ui_setup()
+        self.cal_layer = self.create_or_retrieve_magnification_layer()
```

Comment:
Might be worth renaming that to `self.magnification_layer` whilst we're in a renaming mood.